install/kubernetes/helm/subcharts/gateways/templates/deployment.yaml

```diff
@@ -197,6 +197,14 @@ spec:
             value: "true"
           {{- end }}
           {{- end }}
+          - name: ISTIO_META_WORKLOAD_SDS_ENABLED
```

Comment:
Same comment - if sds is enabled we should not send the meta.

Reply:
updated.